src/main/java/com/singlestore/jdbc/client/column/StringColumn.java

```diff
@@ -75,8 +75,8 @@ public StringColumn useAliasAsName() {
   }
 
   @Override
-  public String defaultClassname(Configuration conf) {
-    return String.class.getName();
+  public String defaultClassname(final Configuration conf) {
+    return isBinary() ? "byte[]" : String.class.getName();
```

Comment:
Hmm... 
Is it possible to use `byte[].class.getName()`

Reply:
No) because primitive array encoded differently